### ISSUE TITLE
mount.zfs is not linked to libselinux.so

### ISSUE THREAD

**Issue by dajhorn** [zfsonlinux/pkg-zfs#10]
Coreutils pre-depends on libselinux1, and `/bin/mount` is linked to `/lib/x86_64-linux-gnu/libselinux.so.1`.

Although `/sbin/mount.zfs` can be selinux aware, the PPA binary is currently missing that linkage:

```
$ ldd /sbin/mount.zfs 
linux-vdso.so.1 =>  (0x00007fffb15ff000)
libspl.so.1 => /lib/libspl.so.1 (0x00007f757e30c000)
libzfs.so.1 => /lib/libzfs.so.1 (0x00007f757e0d6000)
libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f757deb8000)
libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f757db19000)
librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f757d911000)
/lib64/ld-linux-x86-64.so.2 (0x00007f757e537000)
libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f757d68c000)
libefi.so.1 => /lib/libefi.so.1 (0x00007f757d486000)
libnvpair.so.1 => /lib/libnvpair.so.1 (0x00007f757d277000)
libshare.so.1 => /lib/libshare.so.1 (0x00007f757d070000)
libuutil.so.1 => /lib/libuutil.so.1 (0x00007f757ce66000)
libzpool.so.1 => /lib/libzpool.so.1 (0x00007f757cba4000)
libavl.so.1 => /lib/libavl.so.1 (0x00007f757c9a0000)
libuuid.so.1 => /lib/x86_64-linux-gnu/libuuid.so.1 (0x00007f757c79b000)
libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f757c583000)
libunicode.so.1 => /lib/libunicode.so.1 (0x00007f757c332000)
```

**Comment by behlendorf** [comment]
These seems like a generic bug.  We should be linking against $(LIBSELINUX) if it's detected and I see we are not.  The fix on your issue462 branch looks reasonable, I'll give it a try on all the platforms.

**Comment by dajhorn** [comment]
This issue was resolved by commit SHA afd7da0ce72c3b3554079644d73e90fe6d2bf955.
